Neos.Flow/Classes/Security/Authentication/Token/PasswordToken.php

```diff
@@ -42,22 +39,30 @@ class PasswordToken extends AbstractToken
      *
      * @param ActionRequest $actionRequest The current action request
      * @return void
+     * @throws InvalidAuthenticationStatusException
      */
     public function updateCredentials(ActionRequest $actionRequest)
     {
         if ($actionRequest->getHttpRequest()->getMethod() !== 'POST') {
             return;
         }
-
-        $postArguments = $actionRequest->getInternalArguments();
-        $password = ObjectAccess::getPropertyPath($postArguments, '__authentication.Neos.Flow.Security.Authentication.Token.PasswordToken.password');
-
+        $allArguments = array_merge($actionRequest->getArguments(), $actionRequest->getInternalArguments());
```

Comment:
Why would we want to look into internal arguments here? The method states it looks into POST, which would be satisfied by getArguments() alone?!

Reply:
`__authentication.Neos.Flow.Security.Authentication.Token.UsernamePassword.username` _is_ considered an internal argument by Flow...

Reply:
Oh yeah, ofc 🤦 mmmm, tricky that